templates/portfolios/portfolios_table.html

```diff
@@ -28,12 +28,24 @@
     {% get_obj_perms request.user for portfolio as "perms" %}
     {% if "can_grant_portfolio_owner_permission" in perms %}
     <div class="col-md-2">Owner</div>
+        <div class="col-md-2">
+            <span class="glyphicon glyphicon-pencil"></span>&nbsp;&nbsp;&nbsp;
```

Comment:
Should the pencil icon be inside the 'a' tag as a clickable part of the link?

Reply:
If not, it seems like it will get crowded to have multiple unclickable pencil icons in the table. Maybe just have one as a separation from the readable and editable actions on the table, to avoid crowding? Just a thought.

Reply:
So this is an attempt to be consistent with how a user chooses a portfolio to view. They click on the name and not the icon or combination of name & icon. The reason the icon is there is to avoid the user not recognizing that that section is where you can pick a portfolio to edit/delete.

Reply:
Oh okay, I didn't notice the folder icon for viewing the portfolio. In that case, this makes sense to me. I might still suggest a single pencil icon before all the portfolio edit commands, rather than one before each command, to avoid crowding, but it makes sense to me now why it's not clickable, for consistency. Thanks!